src/app/utils/index.js

```diff
@@ -0,0 +1,3 @@
+export function classNames(...classes) {
```

Comment:
There is a pretty awesome `classnames` package which offers a few more use cases. Perhaps it would make sense to use it instead https://www.npmjs.com/package/classnames